services/likes_service.py

```
@@ -2,8 +2,10 @@
 from datetime import datetime,timezone
 from models.like import LikeCreate, LikeResponse
 from repositories.likes_repo import get_likes, save_likes
+from typing import Optional, Annotated, Any, Generator,List
```

Ruff (0.14.13)

Import from `collections.abc` instead: `Generator`

Import from `collections.abc`

(UP035)